Packages/GemAPI/Sources/GemAPI.swift

```diff
@@ -195,10 +195,11 @@ public enum GemAPI: TargetType {
             ].compactMapValues { $0 }
             
             return .params(params)
-        case .getSearchAssets(let query, let chains):
+        case .getSearchAssets(let query, let chains, let tag):
```

Comment:
```suggestion
        case .getSearchAssets(let query, let chains, let tags):
```

it should support multiple tags, for for now only pass one or [], same idea with chains